### ISSUE TITLE
[Bug] Linux Multiplayer Lobby

### ISSUE THREAD

**Issue by Diyou** [yuzu-emu/yuzu#8897]
When opening the Public Room Browser it does not list any lobbies.
The Log says:
```
[  28.870628] WebService <Error> web_service/web_backend.cpp:GenericRequest:114: GET to https://api.yuzu-emu.org/lobby returned null
```
I selected the correct Network Interface.
I'm using manjaro and the ea version 2948.

**Comment by Docteh** [comment]
Try starting the AppImage like `SSL_CERT_FILE=/etc/ssl/cert.pem ./yuzu-whatever.AppImage`

**Comment by Docteh** [comment]
Yeah, if you weren't on manjaro I'd bother you into testing the build artifact for #8871 the Linux option has an appimage
